src/pages/debrief/debrief.ts

```diff
@@ -174,6 +180,14 @@ export class DebriefPage extends BasePageComponent {
     this.store$.dispatch(new DebriefViewDidEnter());
   }
 
+  ionViewDidLeave(): void {
+    if (super.isIos() && this.isPracticeTest) {
+      this.deviceProvider.disableSingleAppMode();
```

Comment:
Not sure if we wanted single app mode enabled? I'd confirm with ammar

Reply:
Yeah, just discussed with Ammar and I'll take it out. We were going to keep things consistent and enable it, but it might just be annoying.